src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 
 CLOUD_PROVIDER=${1:-$CLOUD_PROVIDER}
+# Enabling dynamically loaded cloud provider plugin feature, disabled by default
+CLOUD_PROVIDER_PLUGIN_ENABLE=${CLOUD_PROVIDER_PLUGIN_ENABLE:-false}
+# caa will try to load extend cloud provider from "${CLOUD_PROVIDER_PLUGIN_PATH}/${CLOUD_PROVIDER}.so" in runtime
+# specific env variables for any new extend cloud providers must be set and valid
+CLOUD_PROVIDER_PLUGIN_PATH=${CLOUD_PROVIDER_PLUGIN_PATH:-/cloud-providers}
```

Comment:
Can we change the name of this config to `CLOUD_PROVIDER_EXTERNAL_PLUGINS_PATH`

Reply:
updated.